scripts/tokenomics-emulation/emulations/stash-account-balance.emulation.ts

```diff
@@ -0,0 +1,67 @@
+import { IEmulation } from "./emulation.interface";
+import Network from "../network";
+import fs from "fs";
+
+class StashAccountBalanceEmulation implements IEmulation {
+  constructor(
+    private readonly networkConfig,
+    private readonly network: Network
+  ) {}
+
+  public async run(): Promise<void> {
+    console.log(`Fetch balance of Validators and Nominators Stash account`);
+    const genericValidatorsCount = this.networkConfig.validators.amount;
+    const nominatorsCount = this.networkConfig.nominators.amount;
+    const genesisValidatorsCount = this.networkConfig.genesis_validators_amount;
+
+    console.log(`About to fetch balance of Genesis Validators stash account - ${genesisValidatorsCount}\n`);
```

Comment:
Move common code to the private method and reuse it.

Reply:
Moved to private method